pyenv-win/bin/pyenv.bat

```diff
@@ -11,7 +11,7 @@ if [%1]==[] (
 )
 
 rem use pyenv.vbs to aid resolving absolute path of "active" version into 'bindir'
-for /f %%i in ('%pyenv% version') do call :normalizepath "%~dp0..\versions\%%i" bindir
+for /f %%i in ('%pyenv% vname') do call :normalizepath "%~dp0..\versions\%%i" bindir
```

Comment:
Proper to use `vname` since we only car about versions here (not the files setting them).